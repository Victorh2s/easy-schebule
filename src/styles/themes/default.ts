export const defaultTheme = {
  Light: {
    button: '#5300FF',
    success: '#2AAA50',
    error: '#D42029',
    gray: '#A4A69F',
    'font-color': '#FFFFFF',
    'logo-color': '#05543A',
    'pendente-color': '#F73A4E',
    'fazendo-color': '#FF911C',
    'feito-color': '#2AE850',
    'background-color': '#0f0f0f',
    'link-color': '#15A89F',
    'background-form': '#171717',
    'border-input': '#D2D2D4',
  },
  Dark: {
    button: '#5300FF',
    success: '#2AAA50',
    error: '#D42029',
    gray: '#A4A69F',
    'font-color': '#0f0f0f',
    'logo-color': '#05543A',
    'pendente-color': '#F73A4E',
    'fazendo-color': '#FF911C',
    'feito-color': '#2AE850',
    'background-color': '#FFFFFF',
    'link-color': '#15A89F',
    'background-form': '#EDEDEF',
    'border-input': '#D2D2D4',
  },
} as const;
