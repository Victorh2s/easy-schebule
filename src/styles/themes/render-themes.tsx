import { render, RenderResult } from '@testing-library/react';
import { ThemeProvider } from 'styled-components';
import { defaultTheme } from './default';
import { BrowserRouter } from 'react-router-dom';

export const renderTheme = (children: React.ReactNode): RenderResult => {
  return render(
    <BrowserRouter>
      <ThemeProvider theme={defaultTheme}>{children}</ThemeProvider>{' '}
    </BrowserRouter>
  );
};
