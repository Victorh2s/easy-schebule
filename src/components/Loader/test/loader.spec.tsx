import { IsLoader } from '..';
import { renderTheme } from '../../../styles/themes/render-themes';

it('should match to snapshot', () => {
  const { container } = renderTheme(<IsLoader />);

  expect(container).toMatchSnapshot();
});
