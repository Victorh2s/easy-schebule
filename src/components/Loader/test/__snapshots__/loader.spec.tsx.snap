// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should match to snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  height: 60vh;
}

.c1 {
  width: 50px;
  height: 50px;
  border: 5px solid #5300FF;
  border-top-color: #FFFFFF;
  border-radius: 50%;
  -webkit-animation: cilQsd 1s infinite linear;
  animation: cilQsd 1s infinite linear;
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    />
  </div>
   
</div>
`;
